docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/Plugin/Block/DisqusComments.php

```diff
@@ -118,12 +118,11 @@ private function getCurrEntity() {
   /**
    * Create a new node storage instance.
    *
-   * @return Drupal\Core\Entity\EntityStorageInterface
+   * @return \Drupal\Core\Entity\EntityStorageInterface
```

Comment:
this should have an `|null` at the end if it can be null. Otherwise update the comment.

However, getStorage will not return a null. So please update `The node storage or NULL.` to just be `The node storage.`